src/App.js

```diff
@@ -3,6 +3,7 @@ import Home from './components/Home';
 import Form from './components/Form';
 import Error from './components/Error';
 import NavBar from './components/NavBar';
+import FormError from './components/FormError';
```

Comment:
Importing FormError in Form.jsx should be sufficient FromError is imported here and never used so I would recommend taking it out